### PR TITLE
dmd 2.071.2-b3 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by ilovezfs** [comment]
@John-Colvin I think you read my mind. I was just looking at this tag!

File: Formula/dmd.rb [hunk 1]
```diff
@@ -29,23 +29,23 @@ class Dmd < Formula
   end
 
   devel do
-    url "https://github.com/dlang/dmd/archive/v2.071.2-b1.tar.gz"
-    sha256 "14111e8347a54837e2f06347b1b8a38f644435589a9aa8b3672e6928295a5ad8"
+    url "https://github.com/dlang/dmd/archive/v2.071.2-b3.tar.gz"
+    sha256 "751e306639535dc54b5befd704d72067c174f1190a73be12da22d131a66299f5"
     version "2.071.2-b1"
```
Comment (ilovezfs) [inline Formula/dmd.rb]: this should be updated to say
```
version "2.071.2-b3"
```

Comment (ilovezfs) [inline Formula/dmd.rb]: version also should go below the url and above the sha256.

Comment (John-Colvin) [inline Formula/dmd.rb]: fixed

File: Formula/dmd.rb [hunk 2]
```diff
@@ -29,23 +29,23 @@ class Dmd < Formula
   end
 
   devel do
-    url "https://github.com/dlang/dmd/archive/v2.071.2-b1.tar.gz"
-    sha256 "14111e8347a54837e2f06347b1b8a38f644435589a9aa8b3672e6928295a5ad8"
-    version "2.071.2-b1"
+    url "https://github.com/dlang/dmd/archive/v2.071.2-b3.tar.gz"
+    sha256 "751e306639535dc54b5befd704d72067c174f1190a73be12da22d131a66299f5"
+    version "2.071.2-b3"
```
Comment (ilovezfs) [inline Formula/dmd.rb]: should be moved between the url and the sha256

File: Formula/dmd.rb [hunk 3]
```diff
@@ -29,23 +29,23 @@ class Dmd < Formula
...
 
     resource "druntime" do
-      url "https://github.com/dlang/druntime/archive/v2.071.2-b1.tar.gz"
-      sha256 "3cc3e7bbdf3fbc0b0e9f5d472088921bd2c9f20e39e40dcd47aae4758c59072c"
+      url "https://github.com/dlang/druntime/archive/v2.071.2-b3.tar.gz"
```
Comment (ilovezfs) [inline Formula/dmd.rb]: I know it sounds crazy but you also need to add
```
version "2.071.2-b3"
```
for each of these as well.

File: Formula/dmd.rb [hunk 4]
```diff
@@ -29,23 +29,23 @@ class Dmd < Formula
...
+      sha256 "d444312f483eac0e6b216ba6864fb9132cb999e0a8ebd9f9dc40148d7efb2149"
     end
 
     resource "phobos" do
-      url "https://github.com/dlang/phobos/archive/v2.071.2-b1.tar.gz"
-      sha256 "f9913c9e6653c10c9a0377878dd8f82a921c576af6e1be5620b81ab28deb2dfc"
+      url "https://github.com/dlang/phobos/archive/v2.071.2-b3.tar.gz"
```
Comment (ilovezfs) [inline Formula/dmd.rb]: add
```
version "2.071.2-b3"
```

File: Formula/dmd.rb [hunk 5]
```diff
@@ -29,23 +29,23 @@ class Dmd < Formula
...
+      sha256 "4b9cef3e522455bd55e1fbe58a62152e61bf83d433841caa3750c7fcdd1eab29"
     end
 
     resource "tools" do
-      url "https://github.com/dlang/tools/archive/v2.071.1-b1.tar.gz"
-      sha256 "5f1a9de622f5228b7bbea05597461b0cd167cf9e64d3fe936a04bb02690f5279"
+      url "https://github.com/dlang/tools/archive/v2.071.2-b3.tar.gz"
```
Comment (ilovezfs) [inline Formula/dmd.rb]: add
```
version "2.071.2-b3"
```

**Comment by ilovezfs** [comment]
@John-Colvin I went ahead and made those last couple changes after pulling your PR. Thanks so much for your contribution to Homebrew! :tada:

**Comment by John-Colvin** [comment]
Ah, I'd actually made the changes  but forgot to push. Thanks for sorting it out :)

**Comment by ilovezfs** [comment]
@John-Colvin Ah no problem at all (since it's devel no bottling hoops to jump through to make tweaks in post, at least not until we have devel bottles ...). Thanks again!

**Pull request closed without merging**